lib/widgets/action_sheet.dart

```diff
@@ -169,10 +131,9 @@ class MessageActionSheetCancelButton extends StatelessWidget {
         foregroundColor: designVariables.contextMenuCancelText,
         shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(7)),
         splashFactory: NoSplash.splashFactory,
-      ).copyWith(backgroundColor: WidgetStateColor.fromMap({
-        WidgetState.pressed: designVariables.contextMenuCancelPressedBg,
-        ~WidgetState.pressed: designVariables.contextMenuCancelBg,
-      })),
+      ).copyWith(backgroundColor: WidgetStateColor.resolveWith((states) =>
+          designVariables.contextMenuCancelBg.withFadedAlpha(
+            states.contains(WidgetState.pressed) ? 0.20 : 0.15))),
```

Comment:
Seems like the colors got swapped between the cancel button and the others.

Meanwhile, these colors are slightly changed from the previous colors for the normal buttons: was alpha 0.12, now 0.15. That happens in a commit marked NFC, so it's a bug of one kind or another.

Reply:
Updated the PR to fix that. This came from the push ended with 9c8316f8a02e383ec86d15239dfb6dd6aa6ad334. I think this is an error in resolving conflicts with the [commit](63146d378692635194cc733675d2d964b29f117c) that introduced the `contextMenuCancelBg` theme variable.

Reply:
Cool. Yeah, always good to reread one's branch after a rebase that had conflicts. If you have your Git diff-viewer set up to show moves, then this pair of changes stands out because they're non-moves in the midst of a big move.